src/linux/api.h

```diff
@@ -10,17 +10,19 @@
 
 #define CPUINFO_LINUX_FLAG_PRESENT            UINT32_C(0x00000001)
 #define CPUINFO_LINUX_FLAG_POSSIBLE           UINT32_C(0x00000002)
-#define CPUINFO_LINUX_FLAG_MAX_FREQUENCY      UINT32_C(0x00000004)
-#define CPUINFO_LINUX_FLAG_MIN_FREQUENCY      UINT32_C(0x00000008)
-#define CPUINFO_LINUX_FLAG_SMT_ID             UINT32_C(0x00000010)
-#define CPUINFO_LINUX_FLAG_CORE_ID            UINT32_C(0x00000020)
-#define CPUINFO_LINUX_FLAG_PACKAGE_ID         UINT32_C(0x00000040)
-#define CPUINFO_LINUX_FLAG_APIC_ID            UINT32_C(0x00000080)
-#define CPUINFO_LINUX_FLAG_SMT_CLUSTER        UINT32_C(0x00000100)
-#define CPUINFO_LINUX_FLAG_CORE_CLUSTER       UINT32_C(0x00000200)
-#define CPUINFO_LINUX_FLAG_PACKAGE_CLUSTER    UINT32_C(0x00000400)
-#define CPUINFO_LINUX_FLAG_PROC_CPUINFO       UINT32_C(0x00000800)
-#define CPUINFO_LINUX_FLAG_VALID              UINT32_C(0x00001000)
+#define CPUINFO_LINUX_FLAG_CUR_FREQUENCY      UINT32_C(0x00000004)
+#define CPUINFO_LINUX_FLAG_MAX_FREQUENCY      UINT32_C(0x00000008)
+#define CPUINFO_LINUX_FLAG_MIN_FREQUENCY      UINT32_C(0x00000010)
+#define CPUINFO_LINUX_FLAG_SMT_ID             UINT32_C(0x00000020)
+#define CPUINFO_LINUX_FLAG_CORE_ID            UINT32_C(0x00000040)
+#define CPUINFO_LINUX_FLAG_PACKAGE_ID         UINT32_C(0x00000080)
+#define CPUINFO_LINUX_FLAG_APIC_ID            UINT32_C(0x00000100)
+#define CPUINFO_LINUX_FLAG_SMT_CLUSTER        UINT32_C(0x00000200)
+#define CPUINFO_LINUX_FLAG_CORE_CLUSTER       UINT32_C(0x00000400)
+#define CPUINFO_LINUX_FLAG_CLUSTER_CLUSTER    UINT32_C(0x00000800)
+#define CPUINFO_LINUX_FLAG_PACKAGE_CLUSTER    UINT32_C(0x00001000)
+#define CPUINFO_LINUX_FLAG_PROC_CPUINFO       UINT32_C(0x00002000)
+#define CPUINFO_LINUX_FLAG_VALID              UINT32_C(0x00004000)
```

Comment:
Please do not change the values of existing `CPUINFO_LINUX_FLAG_VALID`, just add new flags at the end

Reply:
Fixed, moved new flags to the end.